src/qt/locale/bitcoin_en.xlf

```diff
@@ -3064,7 +3064,7 @@ If you are receiving this error you should request the merchant provide a BIP21
         <context-group purpose="location"><context context-type="linenumber">1067</context></context-group>
```

Comment:
Please don't update the translation files in a PR, we leave this until specific times to minimize merge collisions.